prime-dotnet-webapi/Controllers/EnrolleePaperSubmissionsController.cs

```diff
@@ -258,5 +268,72 @@ public async Task<ActionResult> FinalizeEnrolleePaperSubmission(int enrolleeId)
 
             return Ok();
         }
+
+        // HEAD: api/Enrollees/potential-paper-enrollee/5
+        /// <summary>
+        /// Gets all paper enrollees and checks whether or not there is a match in dateOfBirth with the current logged on enrollee.
+        /// </summary>
+        [HttpHead("potential-paper-enrollee/{dateOfBirth}")]
```

Comment:
The intention of a HEAD request is that it is equivalent to a GET without returning a body, so I would structure this URL more like a GET with a filter. 

Either
`{enrolleeId}/paper-submissions?dob=<dateOfBirth>` (which I prefer slightly more)
or
`{enrolleeId}/paper-submissions/date-of-birth/<dateOfBirth>` / `{enrolleeId}/paper-submissions/dob/<dateOfBirth>`

Reply:
Fixed